<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EpisodeConverter</class>
 <widget class="QDialog" name="EpisodeConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Episode conversion tool</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>This wizard will convert / repair level and episode files between different formats
Supported formats:
- PGE-X LVLX/WLDX
- SMBX Version 1...64 LVL/WLD
- SMBX-38A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="episodePath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Episode path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="targetFormat">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PGE-X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SMBX-1...64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SMBX-38A</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target format:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="targetFormatVer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>64</number>
   </property>
   <property name="value">
    <number>64</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="lookForSubDirs">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Look for subdirectories</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="makeBacup">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Make backup (&quot;_backup&quot; sub-folder will be created)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DoMadJob">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>220</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
